core/unit_test/CMakeLists.txt

```diff
@@ -219,7 +219,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;HIP;SYCL)
     if (Tag STREQUAL "Cuda")
       set(TagHostAccessible CudaUVM)
     elseif(Tag STREQUAL "HIP")
-      set(TagHostAccessible HIPHostPinned)
+        set(TagHostAccessible HIPManaged)
```

Comment:
Fix the indentation

Reply:
fixed in [f26b9b](https://github.com/kokkos/kokkos/pull/5112/commits/f26b9b74267331d8d606e62baf7bb70d281d909c)